Tests/PlayerTests/Player/PlayerTests.swift

```diff
@@ -8,7 +8,8 @@ import XCTest
 final class PlayerTests: XCTestCase {
 	private var player: Player!
 	private var playerEventSender: PlayerEventSenderMock!
-	private var dbQueue: DatabaseQueue!
+	private var offlineDbQueue: DatabaseQueue!
+	private var cachedDQueue: DatabaseQueue!
```

Comment:
[suggestion] small typo I guess to make it consistent with the above
```suggestion
	private var cachedDbQueue: DatabaseQueue!
```